views/webcam.pug

```diff
@@ -7,18 +7,18 @@ block append stylesheets
     link(rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css')
 block content
     .container-fluid.text-center#webcam
-        .breakSpace
         h2.text-white#header How are you feeling right now?
-        div#videowrap
-            video#video(playsinline='' autoplay='')
+        .container-fluid.d-flex.justify-content-center
+            video#video.video-animation(playsinline='' autoplay='')
             canvas#canvas
-        div#controller
-            button.btn.btn-outline-light#take-photo CAPTURE
-            button.btn.btn-outline-light.mx-2#try-again TAKE PHOTO AGAIN
-            button.btn.btn-outline-light#get-tracks SUBMIT
-    div#loader
-        div#box
-        div#shadow
-        .mt-5
-        h1.text-white#loadingtext Curating your playlist
+        .container-fluid.text-center
+            #beforeCaptureGroup.btn-group.btn-group-lg(role='group')
+                button.btn.btn-outline-light(type='button')#take-photo CAPTURE
+            #afterCaptureGroup.btn-group.btn-group-lg(role='group')
+                button.btn.btn-outline-light.mx-2(type='button')#try-again TAKE PHOTO AGAIN
+                button.btn.btn-outline-light(type='button' data-toggle="tooltip" data-placement="top" title="Submit your photo and we'll get you tracks curated to your mood and Spotify account")#get-tracks SUBMIT
```

Comment:
Thought it would be better for tooltip, since the webcam doesn't have much information, this would help guide the user and feel more comfortable w submitting photo.

![image](https://user-images.githubusercontent.com/49849754/93723120-c1775480-fb50-11ea-8ce6-348c706ab81f.png)

Reply:
Down to include, looks nice :) small spelling mistake!
Maybe change to: "Submit your photo and we'll have your curated tracks in no time!"

Reply:
Hmm I thought it would be nice to tell them that the tracks will be relevant to their mood and Spotify history, cause we mention it in the tutorial/about us, but i thought itd be nice to quickly mention about spotifyHistory, but yah lmk what you think